src/main/metabase/api.ts

```diff
@@ -248,11 +274,15 @@ export async function buildInitialQueries(
   // Picks some fields which aren't AIDs to put in the GROUP BY.
   const nonAidField = fields.find((field) => !aidColumns.includes(field.name));
 
-  const columnSQL = nonAidField ? `"${nonAidField.name}",` : '';
-  const groupBySQL = nonAidField ? 'GROUP BY 1' : '';
-  const query = `SELECT ${columnSQL} count(*) FROM "${tableName}" ${groupBySQL}`;
+  const columnSQL = nonAidField ? `${postgresQuote(nonAidField.name)},` : '';
+  const groupBySQL = nonAidField ? `GROUP BY ${postgresQuote(nonAidField.name)}` : '';
+  const query = safeSQLFormat(`SELECT ${columnSQL} count(*) FROM ${postgresQuote(tableName)} ${groupBySQL}`);
```

Comment:
We are not accepting dynamic user input that we need to format. We can yielded whatever SQL format we want.

What's the advantage compared to something like this?:
```js
[
  `SELECT ${columns}`,
  'FROM ${table}',
  'GROUP BY 1, 2'
].join('\n')
```

Reply:
It looks a bit more like the indented version Paul suggested, which I don't want to implement by hand, but yours also works. I guess we can get rid of the dependency for now and stick to the simple format.